lib/public/views/QcFlags/details/detailsConfiguration.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { frontLink } from '../../../components/common/navigation/frontLink.js';
+import { formatTimestamp } from '../../../utilities/formatting/formatTimestamp.js';
+import { qcFlagTypeColoredBadge } from '../common/qcFlagTypeColoredBadge.js';
+
+/**
+ * Returns the configuration to display a given QC flag details
+ *
+ * @param {RunDetailsModel} qcFlagsDetailsModel the model storing the QC flag details state
+ * @return {object} A collection of rules to display QC flag details
+ */
+export const qcFlagDetailsConfiguration = (qcFlagsDetailsModel) => ({
+    id: {
+        name: 'Id',
+        visible: true,
+    },
+    pass: {
+        name: 'Pass',
+        visible: true,
+        format: () => qcFlagsDetailsModel.dataPass
+            ? qcFlagsDetailsModel.dataPass.match({
+                Success: (dataPass) => frontLink(dataPass.name, 'runs-per-data-pass', { dataPassId: dataPass.id }),
+                Other: () => null,
+            })
+            : qcFlagsDetailsModel.simulationPass.match({
+                Success: (simulationPass) => frontLink(simulationPass.name, 'runs-per-simulation-pass', { simulationPassId: simulationPass.id }),
+                Other: () => null,
+            }),
+    },
```

Comment:
This column should not be in common active column, you should do the same you did for overview

Reply:
applied